shell.nix

```diff
@@ -5,16 +5,15 @@ let
   pkgs = import
     (fetchTarball {
       name = "nixpkgs-unstable-new-enough-pdm";
-      url = "https://github.com/NixOS/nixpkgs/archive/43862987c3cf2554a542c6dd81f5f37435eb1423.tar.gz"; # keep in sync with .github/workflows/lint.yml
-
+      url = "https://github.com/NixOS/nixpkgs/archive/693668dbb00a0ddbe2e381077f8038cadb721684.tar.gz"; # keep in sync with .github/workflows/lint.yml
```

Comment:
keep in sync with .github/workflows/lint.yml !!!

Reply:
ooops